sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample8_ChangeNotification.md

```diff
@@ -23,15 +23,14 @@ Next, it enumerates all events and handles all instances of  `AppConfigurationKe
 ```C# Snippet:AzConfigSample8_ChangeNotification
 public void HandleEventGridNotification(string data)
 {
-    var events = EventGridEvent.Parse(data);
+    var events = EventGridEvent.ParseMany(new BinaryData(data));
```

Comment:
I can't comment, but line 18 needs to be updated too

Reply:
👍